sorted_keys.go

```diff
@@ -42,6 +42,15 @@ func (v *Value) MonotonicMarshal() []byte {
 	return monotonicMarshal(v)
 }
 
+func (v *Value) ReversedMonotonicMarshal() []byte {
```

Comment:
wouldn't `XORedMonotonicMarshal` be a more appropriate name?

Reply:
ok I get it now, it's OK but please add some comment what is it used for (here or in the order by file)